### ISSUE TITLE
GLFW Build on macOS - GNU old-style field designator extension warning

### ISSUE THREAD

**Issue by EmreTech** [natinusala/borealis#155]
Hello, so whenever I build the GLFW build of Borealis on macOS (I never got this warning in Ubuntu), I get the following repeated 50 times:

```
../library/include/borealis/views/button.hpp:135:5: warning: use of GNU old-style field designator extension [-Wgnu-designator]
    disabledBorderColor : "",
    ^~~~~~~~~~~~~~~~~~~~~
    .disabledBorderColor = 
```

Is there a way to silence it?

I'm using macOS Big Sur (11.0.1), meson 0.57.1 from brew, and the latest version of the Xcode Developer utils.

**Comment by natinusala** [comment]
Don't silence it, fix it 😉 

Does replacing `:` with `.` work?

**Comment by EmreTech** [comment]
I'll try that

**Comment by EmreTech** [comment]
It gives a whole new error: 
```
../library/include/borealis/views/button.hpp:55:5: error: use of undeclared identifier 'shadowType'
    shadowType . ShadowType::GENERIC,
    ^
../library/include/borealis/views/button.hpp:56:31: error: expected unqualified-id
    hideHighlightBackground . true,
                              ^
../library/include/borealis/views/button.hpp:56:5: error: use of undeclared identifier 'hideHighlightBackground'
    hideHighlightBackground . true,
    ^
```

**Comment by EmreTech** [comment]
Okay, actually, I'm going to try what the compiler suggested (since it's the compiler)

**Comment by natinusala** [comment]
It's not the right syntax lol, the right syntax is .member = value

**Comment by EmreTech** [comment]
Alright, I fixed it! I'm going to open a Pull Request shortly.

**Comment by EmreTech** [comment]
Fixed in #156
